config/config_test.go

```diff
@@ -18,28 +18,29 @@
 package config_test
 
 import (
+	"aos_updatemanager/config"
 	"io/ioutil"
 	"log"
 	"os"
 	"path"
 	"testing"
-
-	"aos_updatemanager/config"
 )
 
 /*******************************************************************************
  * Vars
  ******************************************************************************/
 
-var cfg *config.Config
-var wrongConfigName = "aos_wrongconfig.cfg"
+var (
+	cfg             *config.Config
+	wrongConfigName = "aos_wrongconfig.cfg"
```

Comment:
Can is be const?

Reply:
added this change to commit related to goconst lit warning fixes